Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -959,6 +959,9 @@ class BrowserViewController: UIViewController {
     }
 
     func openURLInNewTab(url: NSURL?, isPrivate: Bool = false) {
+        if tabManager.selectedTab != nil {
+            screenshotHelper.takeScreenshot(tabManager.selectedTab!)
```

Comment:
Instead of checking for `nil` then force unwrapping, you can use a conditional unwrap:

```swift
if let selectedTab = tabManager.selectedTab {
    screenshotHelper.takeScreenshot(selectedTab)
}
```